diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/StringUtils.kt

```diff
@@ -37,6 +39,11 @@ val KOTLIN = KtTokens.KEYWORDS
  */
 val loggerPropertyRegex = "(?iu)^log(?:ger)?$".toRegex()
 
+/**
+ * @return true if this String includes assignment operator, false otherwise
+ */
+fun String.hasAssignment(): Boolean = contains(REGEX_FOR_ASSIGNMENT_DETECTION.toRegex())
```

Comment:
Won't detection by ASTNode type be more reliable?

Reply:
Ok, replaced `hasAssignment` with `isAssignment` from `kotlin-compiler-embeddable`